lib/utils.js

```diff
@@ -130,9 +130,9 @@ async function getIwdPath(xcodeVersion) {
 // this function launches an instruments test with a default test
 // that immediately passes. In this way we can start a simulator
 // and be notified when it completely launches
-async function quickLaunch (udid) {
+async function quickLaunch (udid,
+    appPath = path.resolve(rootDir, 'node_modules', 'ios-test-app', absoluteTestAppPath.iphonesimulator)) {
```

Comment:
You should be able to do: appPath = absoluteTestAppPath.iphonesimulator , I forgot to change it yesterday (my bug).
It is safer ios-test-app may actually be in a parent module.